helpers/hjoblib.py

```diff
@@ -469,6 +469,8 @@ def _parallel_execute_decorator(
     func_name: str,
     processify_func: bool,
     task: Task,
+    enable_logging: bool,
+    verbose_log: bool,
```

Comment:
Let's update the docstring with this. Since this private you can point to the doc of the public one

```
:param enable_file_logging: see same parameter in ...
...
```